config/config_test.go

```diff
@@ -129,6 +260,51 @@ services:
 	if config.Services[0].Interval != 60*time.Second {
 		t.Errorf("Interval should have been %s, because it is the default value", 60*time.Second)
 	}
+
+	if config.Web.Address != DefaultAddress {
+		t.Errorf("Bind address should have been %s, because it is specified in config", DefaultAddress)
+	}
+
+	if config.Web.Port != DefaultPort {
+		t.Errorf("Port should have been %d, because it is specified in config", DefaultPort)
+	}
```

Comment:
```suggestion
	if config.Web.Address != DefaultAddress {
		t.Errorf("Bind address should have been %s, because it is the default value", DefaultAddress)
	}

	if config.Web.Port != DefaultPort {
		t.Errorf("Port should have been %d, because it is the default value", DefaultPort)
	}
```

Reply:
Fixed. A question who should resolve the status of the review request? Am I supposed to do this or is it the "privilege" of the request? In my professional work all request are to resolve by the requester, that's why I do not touch the status.

Reply:
@mindcrime-ilab the status will automatically resolve itself once the condition starts passing again.

The description for the `services[].alerts[].success-threshold` field should shed some light on this:
> Number of successes in a row before an ongoing incident is marked as resolved

Reply:
Oh sorry - my question seems to be misleading - it did not target the process of incident resolving of failing services in gatus itself, but was targeted against the administrative github process. In particular should I resolve the eg. current conversation if I believe I fixed all issues or is it up to you after verifying?

Reply:
Ah I see, there's no fixed process for that, so it's generally up to the person. If it's something simple, it's usually fine if you resolve it. If it's something a bit more complicated, then leaving it open and pinging the reviewer might be a better idea.